Instruments/microwave_tuning_gui.py

```diff
@@ -92,17 +84,17 @@ def orig_zoom_limits(self):
         # about 0.003 GHz and a minimum of -1.2 dBm. These limits produce
         # a plot with the dip centered
         #
-        # dip shown on slack (https://jmfrancklab.slack.com/archives/CLMMYDD98/p1705090019740609)
-        for ini_val, w in [
-            (
-                (myconfig["uw_dip_center_ghz"] - myconfig["uw_dip_width_ghz"]),
-                self.textbox1,
-            ),
-            (
-                (myconfig["uw_dip_center_ghz"] + myconfig["uw_dip_width_ghz"]),
-                self.textbox2,
-            ),
-        ]:
+        # dip shown on slack (https://jmfrancklab.slack.com/archivec/CLMMYDD98/p1705090019740609)
+        for ini_val, w in zip(
+            [
+                str(int((myconfig["uw_dip_center_ghz"] + j) * 1e6))
```

Comment:
rather than wrapping both with these conversion functions, can do it once inside a list comprehension